### DISCUSSION TITLE
[Idea] Provide a way to handle mutliple modules loading different versions of the same Composer package

### DISCUSSION THREAD

**Discussion by lhall-amphibee** [PrestaShop/PrestaShop#36613] [Q&A] [answered]
I'd like to make a follow-up discussion here concerning the issue discussed here : https://github.com/composer/composer/issues/9180

To my knowledge, there isn't yet a way to handle this and I'm running into this exact problem with my module. What is the prefered way to handle this at the moment ?

**Comment by kpodemski** [comment, marked as answer]
There are some solutions:

https://php-prefixer.com/
https://github.com/humbug/php-scoper

It is not something we have any official recommendation on. Do what suits you.

**Comment by lhall-amphibee** [comment]
OK I see. I was expecting some kind of streamlined solution but that could also work.

Thanks for the hints.
